docker-compose.yml

```diff
@@ -24,6 +24,20 @@ services:
       - static_volume:/app/experimenter/served/
     command: bash -c "/app/bin/wait-for-it.sh kinto:8888 -- python bin/setup_kinto.py;/app/bin/wait-for-it.sh db:5432 -- python /app/manage.py collectstatic --noinput;python /app/manage.py runserver 0:7001"
 
+  yarn-nimbus-ui:
+    image: app:build
+    env_file: .env
+    environment:
+      - CHOKIDAR_USEPOLLING=true
```

Comment:
Yeah this is waht tripped me up last time I tried this, normal HMR uses one port for the web content (3000 in this case) and then a separate port for the websocket, which I think will be randomly generated everytime you start the server normally, or you can pass a flag to lock it to something, so in theory you lock it to a port and then expose that in compose and it should work but I couldn't get it working last time I tried, not sure if it's worth another try here?  This semed to work pretty quickly for me